components/automate-ui/src/app/pages/project/rules/project-rules.component.spec.ts

```diff
@@ -0,0 +1,219 @@
+          inputs: ['visible', 'objectNoun', 'objectName'],
+          outputs: ['close', 'deleteClicked']
+        }),
+        MockComponent({ selector: 'chef-control-menu' }),
+        MockComponent({ selector: 'chef-form-field'}),
+        MockComponent({ selector: 'chef-breadcrumbs'}),
+        MockComponent({ selector: 'chef-breadcrumb', inputs: ['link'] }),
+        MockComponent({ selector: 'chef-tab-selector', inputs: ['value'] }),
+        MockComponent({ selector: 'chef-button', inputs: ['disabled'] }),
+        MockComponent({ selector: 'chef-heading' }),
+        MockComponent({ selector: 'chef-icon' }),
+        MockComponent({ selector: 'chef-input', inputs: ['value'] }),
+        MockComponent({ selector: 'chef-error'}),
+        MockComponent({ selector: 'chef-loading-spinner' }),
+        MockComponent({ selector: 'chef-option' }),
+        MockComponent({ selector: 'chef-page',
+          inputs: [
+            'subheading',
+            'confirm-btn-text',
+            'heading',
+            'disable-confirm',
+            'page-loading',
+            'confirm-loading'
+          ],
+          outputs: ['confirm', 'close']
+        }),
+        MockComponent({ selector: 'chef-page-header' }),
+        MockComponent({ selector: 'chef-subheading' }),
+        MockComponent({ selector: 'chef-table' }),
+        MockComponent({ selector: 'chef-thead' }),
+        MockComponent({ selector: 'chef-tbody' }),
+        MockComponent({ selector: 'chef-tr' }),
+        MockComponent({ selector: 'chef-th' }),
+        MockComponent({ selector: 'chef-td' }),
+        MockComponent({ selector: 'ng-container', inputs: ['hidden'] }),
+        ProjectRulesComponent
+      ],
+      imports: [
+        ReactiveFormsModule,
+        RouterTestingModule,
+        ChefPipesModule,
+        StoreModule.forRoot({
+          router: () => ({
+            state: {
+              url: '/projects/uuid-1/rules',
+              queryParams: {},
+              params: {},
+              fragment: '',
+              path: ['/']
+            },
+            previousRoute: {},
+            navigationId: 0
+          }),
+          projects: projectEntityReducer,
+          rules: ruleEntityReducer
+        })
+      ],
+      providers: [
+        FeatureFlagsService
+      ]
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    jasmine.addMatchers(customMatchers);
+    fixture = TestBed.createComponent(ProjectRulesComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  describe('when creating a new rule', () => {
+    beforeEach(() => {
+      component.project = project;
+      component.rule = <Rule>{};
+      component.isLoading = false;
+      fixture.detectChanges();
+    });
+
+    it('the heading should show <Project Name>: Rule', () => {
+      const heading = `${component.project.name}: Rule`;
+      const compHeading = component.getHeading();
+      expect(heading).toBe(compHeading);
```

Comment:
Short enough that you could inline the `compHeading` variable:
```suggestion
      expect(heading).toBe(component.getHeading());
```
Same comment applies to next couple tests below.